src/helpers/index.tsx

```diff
@@ -93,30 +94,37 @@ function getSohmTokenImage() {
   return <SvgIcon component={SOhmImg} viewBox="0 0 100 100" style={{ height: "1rem", width: "1rem" }} />;
 }
 
-export function getOhmTokenImage(w, h) {
-  h !== null ? (h = `${h}px`) : "32px";
-  w !== null ? (w = `${w}px`) : "32px";
-  return <SvgIcon component={OhmImg} viewBox="0 0 32 32" style={{ height: h, width: w }} />;
+export function getOhmTokenImage(w?: number, h?: number) {
+  const height = h == null ? "32px" : `${h}px`;
+  const width = w == null ? "32px" : `${w}px`;
+  return <SvgIcon component={OhmImg} viewBox="0 0 32 32" style={{ height, width }} />;
 }
 
-export function getTokenImage(name) {
+export function getTokenImage(name: string) {
   if (name === "ohm") return getOhmTokenImage();
   if (name === "sohm") return getSohmTokenImage();
 }
 
-export function setAll(state, properties) {
+// TS-REFACTOR-NOTE: these may have to be any for now...
+export function setAll(state: any, properties: any) {
```

Comment:
Yeah that makes sense, we _should_ have setAll for particular states. or move these out  to know what  the setAll is for.